.bazelrc

```diff
@@ -1,12 +1,17 @@
 # Requires at least bazel 6
 common --enable_bzlmod
 
+build --enable_platform_specific_config
+
 # C++, with warnings mostly turned to 11
-build --cxxopt=-xc++                    --host_cxxopt=-xc++
-build --cxxopt=-std=c++23               --host_cxxopt=-std=c++23
-build --cxxopt=-Wall                    --host_cxxopt=-Wall
-build --cxxopt=-Wextra                  --host_cxxopt=-Wextra
-build --cxxopt=-Wno-unused-parameter    --host_cxxopt=-Wno-unused-parameter
+build:macos --macos_minimum_os=10.15
```

Comment:
Can you move the platform specific options after the general ones ? Also, separate with a newline for easier visiblity.